ReleaseFamilyLockActivity/handler.ts

```diff
@@ -0,0 +1,85 @@
+import { fromEither } from "fp-ts/lib/TaskEither";
+import {
+  fromQueryEither,
+  QueryError
+} from "io-functions-commons/dist/src/utils/documentdb";
+import * as t from "io-ts";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { Context } from "vm";
+import { BonusLeaseModel } from "../models/bonus_lease";
+import { trackEvent, trackException } from "../utils/appinsights";
+import { Failure } from "../utils/errors";
+
+export const ReleaseFamilyLockActivitySuccess = t.type({
+  familyUID: NonEmptyString,
+  kind: t.literal("SUCCESS")
+});
+export type ReleaseFamilyLockActivitySuccess = t.TypeOf<
+  typeof ReleaseFamilyLockActivitySuccess
+>;
+
+export const ReleaseFamilyLockActivityResult = t.taggedUnion("kind", [
+  Failure,
+  ReleaseFamilyLockActivitySuccess
+]);
+export type ReleaseFamilyLockActivityResult = t.TypeOf<
+  typeof ReleaseFamilyLockActivityResult
+>;
+
+export const ReleaseFamilyLockActivityInput = t.type({
+  familyUID: NonEmptyString
+});
+
+type IReleaseFamilyLockActivityHandler = (
+  context: Context,
+  input: unknown
+) => Promise<ReleaseFamilyLockActivityResult>;
+
+export function getReleaseFamilyLockActivityHandler(
+  bonusLeaseModel: BonusLeaseModel,
+  logPrefix = `ReleaseFamilyLockActivity`
+): IReleaseFamilyLockActivityHandler {
+  return async (
+    context: Context,
+    input: unknown
+  ): Promise<ReleaseFamilyLockActivityResult> => {
+    return fromEither(ReleaseFamilyLockActivityInput.decode(input))
+      .mapLeft<QueryError>(err => ({
+        body: readableReport(err),
+        code: "error"
+      }))
```

Comment:
I'd model this as a `INVALID_INPUT` error instead of enforcing the cast to a QueryError

Reply:
fixed